### ISSUE TITLE
Banners on all nav link components

### ISSUE THREAD

**Issue by rohan9024** [rohan9024/ecommerce#133]
Banners on every single page of Men, Women, Shoes, Contributors,  Kids , Sale.

For reference- 
![image](https://github.com/rohan9024/ecommerce/assets/64919642/2f869b09-a8ac-4779-9c64-1efa1d71d1e7)

**Comment by singhcoder694** [comment]
Can you explain the issue bit more as what I guess it you need to replicate given image on every page of your website?? Is it?

**Comment by rohan9024** [comment]
@singhcoder694 In the above image, I have provided you some cards. I want you to implement some unique cards w.r.t. to our website on every single page of Men, Women, Shoes, Contributors, Kids , Sale.

**Comment by singhcoder694** [comment]
Yes, I get it but at what position need to implement cards... And what to write in that... (Description or Advertisment)

**Comment by rohan9024** [comment]
@singhcoder694 it's completely upto you ..... you can take inspiration from the above reference 👆

**Comment by Sami3160** [comment]
> @singhcoder694 it's completely upto you ..... you can take inspiration from the above reference 👆

can i work on this issue please? tell me min number of card you want on each page

**Comment by rohan9024** [comment]
> > @singhcoder694 it's completely upto you ..... you can take inspiration from the above reference 👆
> 
> can i work on this issue please? tell me min number of card you want on each page

@Sami3160 yeah sure go for it. Meanwhile

**Comment by Sami3160** [comment]
> > > @singhcoder694 it's completely upto you ..... you can take inspiration from the above reference 👆
> > 
> > 
> > can i work on this issue please? tell me min number of card you want on each page
> 
> @Sami3160 yeah sure go for it. Talking about the number of cards, it's completely upto you.

heyy, this issue is almost done but on 'contributors' page what cards should i use???

**Comment by rohan9024** [comment]
@Sami3160 Well, we have removed the contributors page
